lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -350,8 +351,14 @@ module InternalConstants
 
     # Whether to call metadata agent to retrieve monitored resource.
     config_param :enable_metadata_agent, :bool, :default => false
-    config_param :metadata_agent_url, :string,
-                 :default => DEFAULT_METADATA_AGENT_URL
+
+    # The URL of Metadata Agent.
+    # If this option is not nil, its value will be used.
```

Comment:
This seems overly verbose. How about something like:
```ruby
# If this option is set, its value is used to contact the Metadata Agent. Otherwise, the value of the STACKDRIVER_METADATA_AGENT_URL environment variable is used. If that is also unset, this defaults to 'http://local-metadata-agent.stackdriver.com:8000'.
```
?

Reply:
Changed.